packages/phone-number-privacy/combiner/package.json

```diff
@@ -24,9 +24,9 @@
     "db:migrate": "NODE_ENV=dev ts-node scripts/run-migrations.ts"
   },
   "dependencies": {
-    "@celo/contractkit": "0.4.12",
```

Comment:
The cases where you have fixed versions, you should probably leave them as they are. Remember that we are actually not managing a "real" semantic versioning.
A version change could require some changes/fixes, until we "normalize" the semantic versioning again.
In the case of the `-dev`, as every package is pointing to master, changing the version is sort of an alias change, but all the compatibilities changes should be addressed before. But this is not the case with fixed versions.

So, I recommend to get back to the old versions. And in the future, if those packages owners want to use newer versions, will probably be a card in the milestone